news/2018-06-17-project-loved-week-of-june-17th.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Project Loved: Week of June 17th"
+date: 2018-06-17 20:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) is back with a fresh new set of candidates for the Loved category! Come check them out, and vote for your favourites!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! This week's round of voting showcases some incredibly difficult beatmaps that are sure to give you a challenge. Across the four gamemodes, we've selected 28 maps, and now it's your turn to decide whether or not these will enter the Loved category. If a map has at least 75% positive votes for osu! or osu!taiko, or 80% for osu!catch or osu!mania, it'll make its way into the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=8)!
+
+Next week, these thresholds will be changing. osu! and osu!taiko will be increasing to 80%, following feedback that we've gotten from you! Both of these modes will also be cutting their candidate count down to five—[Toy](https://osu.ppy.sh/users/2757689) believes that this will ensure that all five candidates are strong picks, and it'll make sure that there's no shortage of awesome maps in the future.
+
+I feel like I'm a broken record writing this in every post, but once again, we unfortunately don't have videos. Our old producer hasn't contacted us in a *long* time, so we're working with a new party to see if we can get those back!
+
+Lastly, as a reminder in case if you don't read the end of this post, be sure to submit suggestions for beatmaps to be added to Loved using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)! All of the nominated beatmaps are chosen from submissions on this form, which you can view in the corresponding [Google sheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit#gid=0).
+
+Let's get started!
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/osu/toaru-shoukoku-no-ohimesama.jpg)](https://osu.ppy.sh/community/forums/topics/762191)
+
+[nao - Toaru Shoukoku no Ohimesama](https://osu.ppy.sh/beatmapsets/370605) by [Settia](https://osu.ppy.sh/users/2445897)  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+Toaru Shoukoku no Ohimesama is the quintessential [Settia](https://osu.ppy.sh/users/2445897) map, featuring ridiculous cross screen jumps overtop of a bombastic eroge song. If you aren't up to the challenge of the 8 star top difficulty, there are 4 lower diffs that vary in difficulty for you to enjoy! It's got some jumps, yo.
```

Comment:
Settia's

Reply:
the name is acting as an adjective here, not possessive